code/modules/jobs/job_types/_job.dm

```diff
@@ -61,6 +61,10 @@
 
 	var/list/mind_traits // Traits added to the mind of the mob assigned this job
 
+	/// Traits added to the liver of the mob assigned this job
+	/// (used for the classic "cops heal from donuts" reaction, among others)
```

Comment:
Check the multi-line comment format for autodocs.
Also don't fail to mention this is a lazy list.